docs/developer-guide/plugins.md

```diff
@@ -1,43 +1,45 @@
 # Writing plugins
 
-```js
-// Abbreviated example:
+Plugins are rules and sets of rules built by the community. We recommend adhering to stylelint's [conventions for writing rules](/docs/developer-guide/rules.md#come-up-with-a-name), including conventions for: names, options, messages, tests and docs. These will help create a consistent user experience.
```

Comment:
Done in https://github.com/stylelint/stylelint/commit/d64c6d8142d9234238ba2412e77fcfff113323e1